### PR TITLE
Spaces to tabs plus previous fixes.

### PR DESCRIPTION
**Pull request by MEXdave1997**
Changed Indentation back to tabs, plus the changes in PR #2

### CODE REVIEW

File: gigantic/parser/model.py [hunk 1]
```diff
@@ -74,15 +84,13 @@ class Archetype(UIResource):  # [HeroArchetypeName RxHeroProvider]
 	lowstaminatriggerpercent = 25.0  # 25.0f
 	lowstaminarecoveredpercent = 30.0  # 30.0f
 
-	def __repr__(self):
-		return self.heroarchetypename
-
 
 class_section_map = {
```
Comment (djdduty) [inline gigantic/parser/model.py]: This section is now the last remaining section that needs to have the indentation corrected.

You're using tabs, but you're one indentation level short on the content and closing bracket.

Comment (djdduty) [inline gigantic/parser/model.py]: Sorry that got misplaced, it was meant to refer to the class_section_map definition.

File: gigantic/parser/model.py [hunk 2]
```diff
@@ -74,15 +84,13 @@ class Archetype(UIResource):  # [HeroArchetypeName RxHeroProvider]
...
-		# 'RxUpgradePathProvider': UpgradePath,
-	}
+	""" Map the Gathered Information. """    
+	'RxHeroProvider': Archetype,
+	# 'RxSummonProvider': SummonProvider,
+	'RxSkillProvider': Skill,
+	'RxSkillUpgradeProvider': SkillUpgrade,
+	# 'RxPassiveUpgradeProvider': PassiveUpgrade,
+	# 'RxUpgradePathProvider': UpgradePath,
+}
```
Comment (djdduty) [inline gigantic/parser/model.py]: Please fix the indentation level of `class_section_map` definition as well.

Comment (MEXdave1997) [reply gigantic/parser/model.py]: will do, update is otw.

**Comment by MEXdave1997** [comment]
Sorry, I removed my fork, due to merge conflicts and poor management of my own fork. Will submit a new PR with proper documentation in the future following the proper styling guide.

**Pull request closed without merging**